src/main/cli/cli.c

```diff
@@ -1383,7 +1385,7 @@ static void cliSerial(const char *cmdName, char *cmdline)
     memcpy(currentConfig, &portConfig, sizeof(portConfig));
 
     cliDumpPrintLinef(0, false, format,
-        portConfig.identifier,
+        serialName(portConfig.identifier, emptyName),
```

Comment:
IMO port should always exist here, so fallback name may be a bit more explicit:
```suggestion
        serialName(portConfig.identifier, "INVALID"),
```